### DISCUSSION TITLE
Hide github-actions from activity

### DISCUSSION THREAD

**Discussion by wizche** [community/community#26076] [Actions]
<p>Is there a way to hide commits related to github actions?</p>
<p>I see lots of commits that actually dont interest me because are generated by actions triggered at time interval.</p>
<p>My activity is literally filled with tons of such messages:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/d6ac1f427b1bd95bb5b258b0f0ffffe8c0199d31.png" data-download-href="/uploads/short-url/uD4WOo3zCrfGA9RMjp80w1f0VXP.png?dl=1" title="image" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/1062939/181095124-bb26558c-ed55-4c06-8a15-49b27759db82.png" alt="image" data-base62-sha1="uD4WOo3zCrfGA9RMjp80w1f0VXP" width="690" height="255" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/d/d6ac1f427b1bd95bb5b258b0f0ffffe8c0199d31_2_690x255.png, https://aws1.discourse-cdn.com/github/optimized/2X/d/d6ac1f427b1bd95bb5b258b0f0ffffe8c0199d31_2_1035x382.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/d/d6ac1f427b1bd95bb5b258b0f0ffffe8c0199d31_2_1380x510.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/d6ac1f427b1bd95bb5b258b0f0ffffe8c0199d31_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1387×514 50.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Any way I can hide this? Otherwise I miss events from other repository which are more interesting to me…</p>
<p>Thanks a lot!</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/wizche">@wizche</a>,</p>
<p>The “<strong>All activity</strong>” section on your personal dashboard typically shows updates from repositories you watch or have starred, and from users you follow.<br>
Normally you can un-watch or un-star the repositories, and un-follow the users to disable showing updates from these repositories and users.<br>
More details, see “<a href="https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/about-your-personal-dashboard#staying-updated-with-activity-from-the-community" rel="nofollow noopener"><strong>About your personal dashboard</strong></a>”.</p>
<p>The “<strong>All activity</strong>” section on your organization dashboard shows all recent activity in the organization, including activity in repositories you’re not subscribed to and of people you’re not following.<br>
Currently we have no a way to disable showing updates from some specified repositories.<br>
More details, see “<a href="https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/about-your-organization-dashboard#staying-updated-with-activity-from-the-organization" rel="nofollow noopener"><strong>About your organization dashboard</strong></a>”.</p>

**Comment by wizche** [comment]
<p>Thanks <a class="mention" href="/brightran">@brightran</a>… Unfollowing was enough to make them disappear</p>

**Comment by BrightRan** [comment]
<p>You’re welcome, <a class="mention" href="/wizche">@wizche</a> .</p>
<p>So, you are on your personal dashboard, and unfollowing the repositories and users can disable showing the updates from these repositories and users.</p>
<p>Glad that my suggestions can help you. If possible, maybe you can mark my above suggestions as solution of this topic, so that other users who have the similar questions can notice these suggestions when they are looking for an answer. Thanks.</p>
